RELEASE/scripts/autoscend/paths/pocket_familiars.ash

```diff
@@ -44,132 +35,24 @@ boolean pokefam_makeTeam()
-		{
-			int choice = choice_matcher.group(1).to_int();
-			if(choice == 0)
-			{
-				choice = choice_matcher.group(2).to_int();
-			}
-
-			if($ints[89, 890, 891, 892, 893, 894, 895, 896, 897, 898, 899, 900, 901, 902, 903, 914] contains choice)
-			{
-				return adv1(loc, -1, option);
-			}
-
-			temp = visit_url("choice.php?pwd=" + my_hash() + "&whichchoice=" + choice + "&option=" + get_property("choiceAdventure" + choice).to_int());
-		}
-		choiceLimiter += 1;
-		if(choiceLimiter > 5)
-		{
-			abort("Choice chain too long or I'm stuck!");
-		}
-	}
-	temp = visit_url("fambattle.php");
-
-	if(!contains_text(temp, "Fight!"))
 	{
 		return false;
 	}
 
-	if(svn_info("Ezandora-Helix-Fossil-branches-Release").revision > 0)
+	if (internalQuestStatus("questL12War") != 1)
 	{
-		auto_log_info("Consulting Ezandora's Helix Fossil script:", "green");
-		boolean ignore = cli_execute("ashq import 'Pocket Familiars'; buffer temp = PocketFamiliarsFight();");
-		if($locations[The Defiled Alcove, The Defiled Cranny, The Defiled Niche, The Defiled Nook] contains my_location())
-		{
-			if(item_amount($item[Evilometer]) > 0)
-			{
-				use(1, $item[Evilometer]);
-			}
-		}
-		cli_execute("auto_post_adv");
-		return true;
-	}
-
-	if(get_property("_pokefamFront") == "")
-	{
-		set_property("_pokefamFront", my_poke_fam(0));
-	}
-	if(get_property("_pokefamMiddle") == "")
-	{
-		set_property("_pokefamMiddle", my_poke_fam(1));
-	}
-	if(get_property("_pokefamBack") == "")
-	{
-		set_property("_pokefamBack", my_poke_fam(2));
-	}
-
-
-	familiar blastFam = to_familiar(get_property("_pokefamBack"));
-	familiar midFam = to_familiar(get_property("_pokefamMiddle"));	
-
-	if(contains_text(temp, "famaction[ult_crazyblast-" + to_int(blastFam) + "]"))
-	{
-		temp = visit_url("fambattle.php?pwd&famaction[ult_crazyblast-" + to_int(blastFam) + "]=ULTIMATE%3A+Spiky+Burst");
-	}
-	else
-	{
-		temp = visit_url("fambattle.php?pwd&famaction[backstab-" + to_int(blastFam) + "]=Backstab");
-	}
-	int action = 1;
-
-
-	while(!contains_text(temp, "<!--WINWINWIN-->"))
-	{
-		if((action & 1) == 1)
-		{
-			temp = visit_url("fambattle.php?pwd&famaction[backstab-" + to_int(midFam) + "]=Backstab");
-		}
-		else
-		{
-			temp = visit_url("fambattle.php?pwd&famaction[backstab-" + to_int(blastFam) + "]=Backstab");
-		}
-		action++;
-		if(contains_text(temp, "dejected and defeated"))
-		{
-			break;
-		}
-		if(action > 40)
-		{
-			abort("Can not win this Pocket Familiars battle!");
-		}
+		return false;
 	}
 
-	if($locations[The Defiled Alcove, The Defiled Cranny, The Defiled Niche, The Defiled Nook] contains my_location())
+	if (get_property("hippiesDefeated").to_int() < 1000 && get_property("fratboysDefeated").to_int() < 1000)
 	{
-		if(item_amount($item[Evilometer]) > 0)
-		{
-			use(1, $item[Evilometer]);
-		}
+		auto_log_info("Doing the wars.", "blue");
+		equipWarOutfit();
+		return warAdventure();
 	}
-	cli_execute("auto_post_adv");
-	return true;
-}
+	return false;
+}
```

Comment:
files need to end in a newline otherwise it sometimes causes random errors (I just ran into this problem earlier actually)

Reply:
Added a new commit that contains a space at the end